src/GameWorldSimulator/NeoServer.Game.Common/Contracts/Creatures/IPlayer.cs

```diff
@@ -61,6 +61,10 @@ public delegate void SendMessageTo(ISociableCreature from, ISociableCreature to,
 
 public delegate void WroteText(IPlayer player, IReadable readable, string text);
 
+public delegate void EquipItem(IPlayer player, IItem item, bool isCheck);
```

Comment:
don't we have these events already on Inventory.cs class?

Reply:
i am removed OnPlayerEquip and OnPlayerDeEquip from events and lua revscript, i remenber we have defined some days ago, item requirements its checked using actual behaviour on items.json, minLevel, vocation, slot and etc..

removed on last commit: https://github.com/OpenCoreMMO/OpenCoreMMO/pull/665/commits/28ddf6853dd4bae8c4b284293a7781147cd24365